test/io/test_pcs_converter.py

```diff
@@ -436,5 +494,5 @@ def test_read_write(self):
             fh.write(pcs_string)
         with open(name, 'r') as fh:
             pcs_new = pcs.read(fh)
-        assert (pcs_new, cs)
+        self.assertTrue(pcs_new, cs)
```

Comment:
@mfeurer Do you have any further ideas:
Fixing that check made it fail. I do not know why.

Looking at the output of print ```pcs_new``` and ```cs``` I could not find a difference.